brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced column -  z index causing overlay.

### Description

- Setting up a z index to 1 resolve the overlay issue.
- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/7c41fad1-f3b3-4e6f-991e-a0779f93e52e?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots

You can check the issue on [this](https://startercopy.com/) site on this Section (What can Starter Copy write for you?)

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop advanced column in next-release
- Add 3 inner child and style > bg color > #fff
- Then drag and drop any block inside a 3 column
- publish the page and check the front. all 3 column blocks are invisible.
- the issue is due to the z-index. Switch to the above branch which solves this issue.

### Checklist:
